frontend/plan/src/components/schedule/MyCircularProgressBar.js

```diff
@@ -7,7 +7,7 @@ export default function MyCircularProgressBar(props) {
     let color;
     const { value } = props;
     switch (true) {
-        case value === 0:
+        case value === 0 || isNaN(value):
```

Comment:
I think we should try to avoid dealing with NaNs. Maybe make the value -1 if it's invalid?

Reply:
done